src/components/LmsConfigurations/index.test.jsx

```diff
@@ -45,6 +55,7 @@ describe('<LmsConfigurations /> ', () => {
   it('get LMS configurations when present', async () => {
     LmsApiService.fetchMoodleConfig.mockResolvedValue(moodleResponse);
     LmsApiService.fetchCanvasConfig.mockResolvedValue(canvasResponse);
+    LmsApiService.fetchBlackboardConfig.mockResolvedValue(blackboardResponse);
```

Comment:
Please move to a separate test, and test that the Blackboard config results are present. The current test seems to only look for the Moodle test results on the actual page.
(Canvas should also be moved to a separate test.)

Reply:
I agree with also testing specifically Blackboard or Canvas.
However, there are a couple things we _should_ be testing with the multiple fetches at once:
1. If one 404's, it doesn't throw an error - particularly if the first call 404's, it's good to see that we still get past that conditional. (I added this in my PR)
2. If we get all 3 (we actually shouldn't outside of internal folks messing about), it still only swaps to 1 form (we don't support multiple LMS configurations, but at the same time we also don't break on it either).
3. Error page when one or more configs fails to be retrieved.

Reply:
So, sorry. I'm not suggesting we not test multiple fetches at once. 
But the bottom of this test only tested that the Moodle config was fetched successfully:
```
expect(wrapper.state().moodleConfig).toEqual(moodleResponse.data.results[0]);
    expect(wrapper.find('MoodleIntegrationConfigForm').props().config).toEqual(wrapper.state().moodleConfig);	    expect(wrapper.find('MoodleIntegrationConfigForm').props().config).toEqual(wrapper.state().moodleConfig)
    ```
    
    I'm saying that in separate tests, we should also test that the Canvas and Blackboard configs were successful.
    The boiler plate of the fetches can go in a `beforeEach`, and there can be a separate test with expects similar to the above to ensure that everything is working correctly.

Reply:
I'm saying that in separate tests, we should add similar expects for the Canvas and Blackboard configs.
We can do a `beforeEach` for the setup so we don't have to repeat a bunch of code.

Reply:
If while we're at it we can also add an `expect` that tests what the user is seeing rather than simply that state and props have been properly updated, that would also be great. Enzyme makes it pretty easy to test state and props; but the React community is kind of moving away from that in favor of testing actual UI output, which I'm in favor of.

Reply:
That all makes sense and is good to add. The UI stuff I've had some issues trying to implement (submit doesn't behave as expected in a testing environment for some reason), however I've been able to still utilize finds for components. Will have this up soon for you to review and Alex to base off/crib.